staging/src/k8s.io/component-base/metrics/shared/restclient/BUILD

```diff
@@ -0,0 +1,32 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
what does "shared" mean in the package path? It is a shared library obviously. But do we need that in the path?

Reply:
Sorry that was out of date.